tests/tables-test.sps

```diff
@@ -241,7 +241,7 @@
       '(#f #f #t #t #t #t #f #f #f #f #f #f #f #f #f #f))
 
 (test (map hash-table-mutable? test-tables)
-      '(#t #t #t #t #t #t #t #t #t #t #t #t #t #t #t #f))
+      '(#t #f #t #t #t #t #t #t #t #t #t #t #t #t #t #f))
```

Comment:
The second hash table is created with `hash-table` using the default #f value for mutable, hence it is not mutable.